bigbluebutton-html5/imports/ui/components/loader/component.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
```

Comment:
We didn't, as the link I sent above says - JSX requires React to be in the same scope (for now).
I meant similar cases where we don't import the whole thing, but import just what we want specifically.